python/ray/tune/tune.py

```diff
@@ -455,6 +458,17 @@ def _handle_execute_queue():
             )
         result_buffer_length = 1
 
+    # If reuse_actors is unset, default to False for string and class trainables,
+    # and default to True for everything else (i.e. function trainables)
+    if reuse_actors is None:
+        reuse_actors = not (
+            isinstance(run_or_experiment, str)
+            or (
+                inspect.isclass(run_or_experiment)
+                and issubclass(run_or_experiment, Trainable)
```

Comment:
Can we actually make this a util function? `is_function_trainable` or something.